lighthouse-core/audits/valid-source-maps.js

```diff
@@ -0,0 +1,158 @@
+/**
+ * @license Copyright 2020 Google Inc. All Rights Reserved.
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License. You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
+ */
+'use strict';
+
+const thirdPartyWeb = require('../lib/third-party-web.js');
+const Audit = require('./audit.js');
+const i18n = require('../lib/i18n/i18n.js');
+
+// TODO: web.dev docs, write description
+const UIStrings = {
+  /** Title of a Lighthouse audit that provides detail on HTTP to HTTPS redirects. This descriptive title is shown to users when HTTP traffic is redirected to HTTPS. */
+  title: 'Page has valid source maps',
+  /** Title of a Lighthouse audit that provides detail on HTTP to HTTPS redirects. This descriptive title is shown to users when HTTP traffic is not redirected to HTTPS. */
+  failureTitle: 'Missing source maps for large first-party JavaScript',
+  /** Description of a Lighthouse audit that tells the user that their JavaScript source maps are invalid or missing. This is displayed after a user expands the section to see more. No character length limits. 'Learn More' becomes link text to additional documentation. */
+  description: 'Source maps translate minified code to the original source code. This helps ' +
+    'developers debug in production. In addition, Lighthouse is able to provide further ' +
+    'insights. Consider deploying source maps to take advantage of these benefits. ' +
+    '[Learn more](https://developers.google.com/web/tools/chrome-devtools/javascript/source-maps).',
```

Comment:
This link is fine for now. I have a task to write an article on web.dev describing all the features we can do with source maps, and I think ideally this audit would link to it.

Reply:
Sounds good